.github/workflows/pr-preview.yml

```diff
@@ -131,8 +131,13 @@ jobs:
         timeout-minutes: 120
         run: |
           sudo apt update
-          sudo apt install rsync
+          sudo apt install rsync jq
           make package
+          INDEX_NAME=$(jq '."index.html".name' lib/streamlit/static/manifest.json)
+          if [ "${INDEX_NAME}" != "index-testFailure" ]:
```

Comment:
Intentionally setting this to verify that this test fails for now, then will switch this to `"${INDEX_NAME}" = "index"` once that's done.